contracts/scripts/native_solc_compile_all_functions

```diff
@@ -31,15 +31,15 @@ compileContract () {
 # Version 0 (Testnet Beta)
 ##########################
 
-compileContract 0_0_0 dev/0_0_0/Functions.sol
```

Comment:
Why do these ones have a `v` added in two places, but the v1 ones below only in one place?

Reply:
Thanks! Fixed